drivers/peci/peci_handlers.c

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <drivers/peci.h>
+#include <syscall_handler.h>
+
+Z_SYSCALL_HANDLER(peci_config, dev, bitrate)
+{
+	return z_impl_peci_config((struct device *)dev, (u16_t)bitrate);
```

Comment:
This isn't correct. You need to validate the device struct being passed in with Z_SYSCALL_DRIVER_PECI().